LethalAPI.TerminalCommands/TerminalNodeExtensions.cs

```diff
@@ -0,0 +1,189 @@
+﻿using LethalAPI.TerminalCommands.Models.Enums;
+using UnityEngine;
+using UnityEngine.Video;
+
+namespace LethalAPI.TerminalCommands
+{
+	/// <summary>
+	/// Provides a collection of Builder extension methods for <seealso cref="TerminalNode"/>
+	/// </summary>
+	public static class TerminalNodeExtensions
+	{
+		/// <summary>
+		/// Sets the display/response text of a terminal node
+		/// </summary>
+		/// <param name="node">Terminal node to modify</param>
+		/// <param name="displayText">New display text</param>
+		/// <returns>Original terminal node</returns>
+		public static TerminalNode WithDisplayText(this TerminalNode node, object displayText)
```

Comment:
Can we use a more descriptive type than object in this param? Or is this the only option?

Reply:
I've re-worded the code docs for this one